### PR TITLE
updated tracking with content grouping

### PR DESCRIPTION
**Pull request by kaylima**
This updates the Google Analytics tracking code to create a new content group (contentGroup1). This new content group will contain data for all content tagged to the Health & Wellness category, and the Health & Wellness category archive page.

### CODE REVIEW

File: wp-content/themes/rivard-report/functions.php [hunk 1]
```diff
@@ -178,7 +178,32 @@ function rivard_popmake_js() {
...
-		error_log(var_export( 'much wow', true));
 	}
 }
 add_action( 'wp_enqueue_scripts', 'rivard_popmake_js' );
+
+function rr_google_analytics() {
+	if ( ! is_user_logged_in() ) { // don't track logged in users ?>
+		<script>
+			( function ( i, s, o, g, r, a, m ) {i['GoogleAnalyticsObject']=r;i[r]=i[r]|| function() {( i[r].q=i[r].q||[] ).push( arguments )},i[r].l=1*new Date();a=s.createElement( o ), m=s.getElementsByTagName( o )[0];a.async=1;a.src=g;m.parentNode.insertBefore( a, m )} )
+			( window,document,'script','https://www.google-analytics.com/analytics.js','ga' );
```
Comment (benlk) [inline wp-content/themes/rivard-report/functions.php]: Reviewing this on vagrant, it looks like we're running both Google Analytics and Universal Analytics on the site at once. Is that something we're worried about?

File: wp-content/themes/rivard-report/functions.php [hunk 2]
```diff
@@ -178,7 +178,32 @@ function rivard_popmake_js() {
...
+					echo "ga( 'set', 'contentGroup1', 'health-wellness' );\n";
+				}
+			} elseif ( is_tax() || is_archive() ) {
+				$term = $wp_query->get_queried_object();
+				if ( ! empty( $term ) && isset( $term->term_id ) && 15314 == $term->term_id ) {
+					echo "ga( 'set', 'contentGroup1', 'health-wellness' );\n";
+				}
+			}
+			?>
+			ga( 'send', 'pageview' );
```
Comment (benlk) [inline wp-content/themes/rivard-report/functions.php]: This chunk looks good.

**Comment by benlk** [comment]
@kaylima at this time, should we bump Largo on Rivard to 0.5.5.4?

**Comment by kaylima** [comment]
> @kaylima at this time, should we bump Largo on Rivard to 0.5.5.4?

I don't see why not; @julia67 any objections?

File: wp-content/themes/rivard-report/functions.php (hunk 1 above)
Comment (kaylima) [reply wp-content/themes/rivard-report/functions.php]: We're only worried if there are 2 or more instances of the old Google Analytics tracking code. Multiple Universal codes and one of the previous kind of code is fine.

**Comment by benlk** [comment]
Largo is updated to 0.5.5.4 in the `master` branch now.

**Pull request merged**